.github/workflows/yem-trigger-monitoring.yaml

```diff
@@ -6,34 +6,48 @@ name: yem-trigger-monitoring
 on:
   workflow_dispatch:
   schedule:
-    - cron: '25 12 * * *'
+    - cron: '10 09 * * *' # chirps-gefs should be updated 9AM UTC, let's give an extra 10 min
```

Comment:
Ah did we confirm that the timestamps [here](https://data.chc.ucsb.edu/products/EWX/data/forecasts/CHIRPS-GEFS_precip_v12/daily_16day/2023/05/23/) are PST and not UTC?

Reply:
yes they are PST